vendor/curl/README.md

```diff
@@ -0,0 +1,69 @@
+# odin-curl
+
+A minimal libcurl binding for the Odin programming language.
+
+## Requirements
+
+- Odin compiler
+- libcurl development files
+  - Ubuntu/Debian: `sudo apt install libcurl4-dev`
+  - Fedora: `sudo dnf install libcurl-devel`
+  - macOS: `brew install curl`
+  - Windows: Download from [curl](https://curl.se/windows/)
```

Comment:
For vendor, things should "just work", especially on Windows. So why does Windows not bundle with a `.lib` already?